### ISSUE TITLE
Miners using the same mine

### ISSUE THREAD

**Issue by GrifterMage** [TangoTek/TekTopia-Community#68]
Saw multiple miners enter the same mine and start mining. One broke the top block, the other broke the bottom block, and then both froze in place.

![2019-05-26_16 42 20](https://user-images.githubusercontent.com/49353797/58388580-9836ca80-7fd5-11e9-8d0a-c495158846fe.png)
![2019-05-26_16 42 59](https://user-images.githubusercontent.com/49353797/58388581-98cf6100-7fd5-11e9-833c-c810b9da65a5.png)

I suspect this may be caused by a race condition related to #50--the second miner was walking right behind the first, so maybe server lag meant that by the time the 'claim' from the first miner went through, the second had already started claiming, so that went through too?

This happened on What If 2 in "Darelko's Village", at about 16:42 Tango Time.

**Comment by JohnyDL** [comment]
Yeah I was thinking that, I've had a few mine from outside the mine when it's short, and others enter a mine together but usually one will reverse course, I've not seen 2 mining any distance together

**Comment by IridescentShadow** [comment]
I've noticed this as well when they are starting new shafts, but it seems to resolve as they get deeper

**Comment by JohnyDL** [comment]
I've found that when this happens near the time a mine is about to exit a village's path finding grid the miners will push each other past the grid's limit and so get trapped there unable to get home

**Comment by tomerorian** [comment]
Happened to me too. in my case it seems it also causes them to bork out and stop working sometimes
